installscripts/jazz-terraform-unix-noinstances/acl.tf

```diff
@@ -1,6 +1,6 @@
 resource "aws_rds_cluster" "casbin" {
   cluster_identifier      = "${var.envPrefix}-${var.acl_db_name}-cluster"
-  availability_zones      = ["us-east-1a", "us-east-1b"]
+  availability_zones      = ["${var.region}a", "${var.region}b"]
```

Comment:
@rajeevr2715 Is there a better way to get the supported AZs for a given region instead of hardcoded suffixes?

Reply:
Travis check is failing too, can you take a look?

Reply:
Yup, looks like this will do it: https://www.terraform.io/docs/providers/aws/d/availability_zones.html

Reply:
@devsatishm Updated code to fetch zones list dynamically. @bleggett Thanks for the link. Seems the travis is checking for python3, ie why its failing.

Reply:
None of the travis config or scripts have changed in the last 4 months, so unless Travis changed their default Python version from 2 to 3 with the news of the upcoming deprecation (which seems doubtful, none of their docs suggest they default to python3) it's something else.

You could try explicitly specifying a Python2 version in `travis.yml` to see if that helps, we can change it back to python3 when the rewrite is merged.

Reply:
@bleggett  Yep, looks like that's what it is - https://travis-ci.com/tmobile/jazz-installer/builds/108580901 

They have put in this - ` This job ran using the default Python version, which was updated to 3.6 on April 16th, 2019. You can explicitly stay on the previous version by specifying python: 2.7 in your .travis.yml.`